<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1386</width>
    <height>836</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UAR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>amplituda</string>
      </property>
     </widget>
    </item>
    <item row="0" column="14" colspan="4">
     <widget class="QCustomPlot" name="sterowanie" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="17">
     <widget class="QSpinBox" name="interwalSpinBox">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="0" column="18" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Wejście</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="zadaneLabel">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Wyjście</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="wyjscieLabel">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="aktualizujButton">
         <property name="text">
          <string>AKTUALIZUJ</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>RESET</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="RoleComboBox">
        <item>
         <property name="text">
          <string>Regulator</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model ARX</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipLineEdit">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="networkModeCheckBox">
        <property name="text">
         <string>Tryb sieciowy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="connectionStatusLabel">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wymianaDanychLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>kp</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="kpLabel">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="11">
     <widget class="QDoubleSpinBox" name="okresLabel">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>40.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QLineEdit" name="zakloceniaLabel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="16">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>interwał [ms]</string>
      </property>
     </widget>
    </item>
    <item row="2" column="14">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>cykl</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLineEdit" name="delayLabel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="tiLabel">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="vecbLabel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>zaklocenia</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="3" column="14">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>czas aktywacji</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="13">
     <widget class="QCustomPlot" name="zadana" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="zmienARX">
      <property name="text">
       <string>Modyfikuj ARX</string>
      </property>
     </widget>
    </item>
    <item row="3" column="15">
     <widget class="QSpinBox" name="aktywacjaLabel">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="tdLabel">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>delay</string>
      </property>
     </widget>
    </item>
    <item row="2" column="16">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>typ sygnalu</string>
      </property>
     </widget>
    </item>
    <item row="1" column="14" colspan="4">
     <widget class="QCustomPlot" name="uchyb" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="vecaLabel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="liczenieCalkiLabel">
      <property name="text">
       <string>Stała całkowania pod całką</string>
      </property>
     </widget>
    </item>
    <item row="3" column="11">
     <widget class="QDoubleSpinBox" name="amplitudaLabel">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="15">
     <widget class="QDoubleSpinBox" name="cyklLabel">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="17">
     <widget class="QComboBox" name="sygnalcomboBox"/>
    </item>
    <item row="2" column="10">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>okres</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>td</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ti</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="resetI">
      <property name="text">
       <string>Wyczyść I</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
